setup.py

```diff
@@ -33,8 +33,9 @@
       author_email='engineering.sophia@swisscom.com',
       license='Apache License 2.0',
       packages=find_packages(exclude=['cleanerversion', 'cleanerversion.*']),
-      url='https://github.com/swisscom/cleanerversion',
+      url='https://github.com/boydjohnson/cleanerversion',
```

Comment:
Just wondering: what's the purpose of changing this URL? I think it's ok if I change it back..